packages/p2p/src/components/order-details/order-details.jsx

```diff
@@ -188,26 +212,110 @@ const OrderDetails = observer(({ onPageReturn }) => {
+                                    label={
+                                        <Text size='xs' weight='bold'>
+                                            {labels.contact_details}
+                                        </Text>
+                                    }
+                                    value={contact_info || '-'}
+                                />
+                            </React.Fragment>
+                        )}
+                        {is_active_order && (
+                            <React.Fragment>
+                                <MyProfileSeparatorContainer.Line className='order-details-card--line' />
+                                <OrderInfoBlock
+                                    className='order-details-card--padding'
+                                    label={
+                                        <Text size='xs' weight='bold'>
+                                            {labels.instructions}
+                                        </Text>
+                                    }
+                                    value={advert_details.description.trim() || '-'}
+                                />
+                            </React.Fragment>
+                        )}
+                        {is_completed_order && !review_details && (
+                            <React.Fragment>
+                                <RatingModal
+                                    is_buy_order_for_user={is_buy_order_for_user}
+                                    is_rating_modal_open={order_store.is_rating_modal_open}
+                                    onClickDone={() => order_store.setOrderRating(id)}
+                                    onClickNotRecommended={() => order_store.setIsRecommended(0)}
+                                    onClickRecommended={() => order_store.setIsRecommended(1)}
+                                    onClickSkip={() => order_store.setIsRatingModalOpen(false)}
+                                    onClickStar={order_store.handleRating}
+                                    rating_value={order_store.rating_value}
+                                />
+                                <div className='order-details-card--rating'>
+                                    <UserRatingButton
+                                        button_text={
+                                            is_reviewable ? localize('Rate this transaction') : localize('Not rated')
+                                        }
+                                        is_disabled={!is_reviewable}
+                                        large
+                                        onClick={() => order_store.setIsRatingModalOpen(true)}
+                                    />
+                                </div>
+                                <Text color='less-prominent' size='xxxs'>
+                                    {!is_reviewable && (
```

Comment:
if the text is empty, would there be an empty space here?

Reply:
I had checked this, it was fine, do you wanna check the test link too?

Reply:
its fine for me 👍